### PR TITLE
Added regex_findall to blitz doc, fixed typos

### PR DESCRIPTION
**Pull request by domachad**
Added documentation to new Blitz enhancement supporting regex_findall while saving an output to a variable
Related to the following PR (in the internal repository): 
https://wwwin-github.cisco.com/pyATS/genielibs/pull/592

### CODE REVIEW

File: docs/blitz/design/save/filters.rst [hunk 1]
```diff
@@ -80,6 +80,28 @@ Below you can see an example of regex filter.
         bootflash:
           value: "The bootflash is %VARIABLES{bootflash} and %VARIABLES{measure}"
 
+Save with Regex findall
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Below you can see an example of regex_findall.
+In this example, execute_output would contain a list of strings such as: 
+['172.16.1.250', '10.12.90.1', '10.12.110.1', '10.12.115.1', '10.12.120.1']
```
Comment (tahigash) [inline docs/blitz/design/save/filters.rst:88]: please update addresses like:
```suggestion
['172.16.1.254', '10.1.1.1', '10.2.2.2', '10.3.3.3', '10.4.4.4']

```

File: docs/blitz/design/save/filters.rst [hunk 2]
```diff
@@ -80,6 +80,28 @@ Below you can see an example of regex filter.
...
+
+    # first saving list of values from execute action output
+    # later on printing this list
+
+    - execute:
+        device: PE1
+        command: show ip interface brief
+        save:
+        - variable_name: execute_output
+          regex_findall: (\d+\.\d+\.\d+\.\d+)                            # returns a list of IP addresses
```
Comment (tahigash) [inline docs/blitz/design/save/filters.rst:100]: maybe too much spaces?
```suggestion
          regex_findall: (\d+\.\d+\.\d+\.\d+)   # returns a list of IP addresses

```

File: docs/blitz/design/save/filters.rst [hunk 3]
```diff
@@ -80,6 +80,28 @@ Below you can see an example of regex filter.
...
+    - print:
+        item:
+          value: "%VARIABLES{execute_output}"
```
Comment (tahigash) [inline docs/blitz/design/save/filters.rst:103]: this is needed?

File: docs/blitz/design/save/index.rst [hunk 4]
```diff
@@ -25,9 +25,11 @@ There are different ways to save values to a variable and also different ways to
...
 
-* :ref:`Regex filter<filters>`: For actions that has string outputs you can apply a regex filter. If regex matches the output, the grouped value, that has a variable name specified like ``(?P<variable_name>)``, will be stored into that variable_name. Below you can find related examples.
+* :ref:`Regex filter<filters>`: For actions that have string outputs you can apply a regex filter. If regex matches the output, the grouped value, that has a variable name specified like ``(?P<variable_name>)``, will be stored into that variable_name. Below you can find related examples.
 
-* :ref:`List filter<filters>`: It is a specific filter that only can be applied on action outputs that are a list.
+* :ref:`Regex findall<filters>`: For actions that have string outputs you can apply a regex findall. It returns a list of all matches in the output, or an empty list if no match is found. The list will be stored into a variable_name previously declared using `variable_name` argument.
```
Comment (tahigash) [inline docs/blitz/design/save/index.rst:30]:
```suggestion
* :ref:`Regex findall<filters>`: For actions that have string outputs you can apply a regex findall filter. It returns a list of all matches in the output, or an empty list if no match is found. The list will be stored into a variable_name previously declared using `variable_name` argument.

```

**Review by tahigash — APPROVED** [review]
approved with comment

File: docs/blitz/design/save/filters.rst (hunk 3 above)
Comment (domachad) [reply docs/blitz/design/save/filters.rst:103]: not really, but I was following the other examples

**Review by lsheikal — APPROVED** [review]
looks good!!! approved

**Pull request merged**